validator_client/src/attestation_service.rs

```diff
@@ -430,6 +430,11 @@ impl<T: SlotClock + 'static, E: EthSpec> AttestationService<T, E> {
             .flatten()
             .unzip();
 
+        if attestations.len() == 0 {
+            warn!("WARN: No attestations were published");
```

Comment:
you need to pass a `log` argument to `warn!` (see other uses in this file)

Reply:
> you need to pass a `log` argument to `warn!` (see other uses in this file)

added a log argument, but accidentally pushed it from my another github acc. sorry for that. I don't know how it happened.